tests/trx_generator/trx_provider.hpp

```diff
@@ -18,45 +19,88 @@ namespace eosio::testing {
       eosio::chain::transaction_id_type _trx_id;
       fc::time_point _sent_timestamp;
 
-      logged_trx_data(eosio::chain::transaction_id_type trx_id, fc::time_point sent=fc::time_point::now()) :
+      explicit logged_trx_data(eosio::chain::transaction_id_type trx_id, fc::time_point sent=fc::time_point::now()) :
          _trx_id(trx_id), _sent_timestamp(sent) {}
    };
 
    struct provider_base_config {
-      std::string _peer_endpoint = "127.0.0.1";
-      unsigned short _port = 9876;
+      std::string    _peer_endpoint_type = "p2p";
+      std::string    _peer_endpoint      = "127.0.0.1";
+      unsigned short _port               = 9876;
 
       std::string to_string() const {
          std::ostringstream ss;
-         ss << "peer_endpoint: " << _peer_endpoint << " port: " << _port;
+         ss << "endpoint type: " << _peer_endpoint_type << "peer_endpoint: " << _peer_endpoint << " port: " << _port;
```

Comment:
Consider some space?
```suggestion
         ss << "endpoint type: " << _peer_endpoint_type << " peer_endpoint: " << _peer_endpoint << " port: " << _port;
```

Reply:
Addressed: [28e8d55](https://github.com/AntelopeIO/leap/pull/1138/commits/28e8d5518991526b66ce68411d1bd309fd3498d7)